cmd/workload/filtertest.go

```diff
@@ -206,14 +212,10 @@ func (fq *filterQuery) run(client *client, historyPruneBlock *uint64) {
 		Addresses: fq.Address,
 		Topics:    fq.Topics,
 	})
-	if err != nil {
-		if err = validateHistoryPruneErr(fq.Err, uint64(fq.FromBlock), historyPruneBlock); err == errPrunedHistory {
```

Comment:
crap.  big oversight on my part.. whoops

Reply:
It was correct at some point.  I must have inadvertently broken it before my original PR was merged.